services/src/ogc/wfs/request.rs

```diff
@@ -44,32 +59,52 @@ pub struct DescribeFeatureType {
     pub output_format: String, // TODO
```

Comment:
Do you know… TODO… what?

Reply:
make an enum out of it or parse it or... do anything with it. Currently we don't use this at all. We don't support this API call. Initially I still mapped them as a placeholder but I can also delete them.

Reply:
Could you add it then?

Reply:
I don't see the benefit. I'd rather delete it if it really bothers you.